scripts/syslog-ng-update-virtualenv.in

```diff
@@ -82,6 +82,8 @@ echo "Running python -m venv..."
 ${SYSTEM_PYTHON} -m venv ${python_venvdir}
 
 echo "Running python -m pip install..."
+${VENV_PYTHON} -m pip install --upgrade pip
+${VENV_PYTHON} -m pip install --upgrade pip setuptools
```

Comment:
This script is executed as a post-install script in our Debian packaging.
I think we should not self-upgrade pip here. We can do that in our Docker images or in the CI instead.

Reply:
It only upgrades the pip in the venv I believe.

Reply:
```
[14:31][ ~/repos/syslog-ng ] (devshell-fix-externally-managed-python) $ python -m venv /tmp/foo
[14:37][ ~/repos/syslog-ng ] (devshell-fix-externally-managed-python) $ pip --version
pip 22.0.2 from /usr/lib/python3/dist-packages/pip (python 3.10)
[14:37][ ~/repos/syslog-ng ] (devshell-fix-externally-managed-python) $ /tmp/foo/bin/python -m pip --version
pip 22.0.2 from /tmp/foo/lib/python3.10/site-packages/pip (python 3.10)
[14:37][ ~/repos/syslog-ng ] (devshell-fix-externally-managed-python) $ /tmp/foo/bin/python -m pip install --upgrade pip
Requirement already satisfied: pip in /tmp/foo/lib/python3.10/site-packages (22.0.2)
Collecting pip
  Using cached pip-23.0.1-py3-none-any.whl (2.1 MB)
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 22.0.2
    Uninstalling pip-22.0.2:
      Successfully uninstalled pip-22.0.2
Successfully installed pip-23.0.1
[14:37][ ~/repos/syslog-ng ] (devshell-fix-externally-managed-python) $ /tmp/foo/bin/python -m pip --version
pip 23.0.1 from /tmp/foo/lib/python3.10/site-packages/pip (python 3.10)
[14:37][ ~/repos/syslog-ng ] (devshell-fix-externally-managed-python) $ pip --version
pip 22.0.2 from /usr/lib/python3/dist-packages/pip (python 3.10)
```

Reply:
We are allowed to do this, right?

Reply:
I think that's okay.